css/css-contain/contain-layout-001.html

```diff
@@ -11,9 +11,14 @@
 <style>
 span {
   contain: layout;
+  display: inline-block;
   height: 100vh; /*If layout containment applies, the span becomes a BFC, height applies, and knocks SS off the page */
 }
+
+#ss {
+  vertical-align: bottom;
```

Comment:
I disagree with Gérard that this is necessary, as the initial value is baseline, which should be sufficient. 
That said, `vertical-align: bottom` isn't wrong, and maybe it is more explicit, so OK.

Reply:
Ok.